config/registry.go

```diff
@@ -58,9 +58,9 @@ var (
 )
 
 var skipList = []string{
-	"aws_waf_rule_group$",              // Too big CRD schema
-	"aws_wafregional_rule_group$",      // Too big CRD schema
-	"aws_mwaa_environment$",            // See https://github.com/crossplane-contrib/terrajet/issues/100
+	"aws_waf_rule_group$",         // Too big CRD schema
+	"aws_wafregional_rule_group$", // Too big CRD schema
+	// "aws_mwaa_environment$",            // See https://github.com/crossplane-contrib/terrajet/issues/100
```

Comment:
If the problem is resolved and we can add this resource, we can remove this line.